src/tests/test_utils/unit_test_utils/mocks/openvino/runtime/mock_iplugin.hpp

```diff
@@ -40,6 +40,11 @@ class MockIPlugin : public ov::IPlugin {
                 import_model,
                 (std::istream&, const ov::SoPtr<ov::IRemoteContext>&, const ov::AnyMap&),
                 (const));
+    MOCK_METHOD(std::shared_ptr<ov::ICompiledModel>, import_model, (std::istream&, std::shared_ptr<ov::AlignedBuffer>, const ov::AnyMap&), (const));
```

Comment:
why not string + Tensor ?

Reply:
This is internal plugins api. All plugins support model import with std::istream. I just added ov::AlignedBuffer as additional argument to simplify implementation to plugins. They can now reuse existing implementation + add additional login to get weights directly from ov::AlignedBuffer